app/screens/RosterScreen/TeamRosterScreen.tsx

```diff
@@ -4,27 +4,75 @@ import { NativeStackScreenProps } from "@react-navigation/native-stack"
 import { Screen } from "@/components/Screen"
 import { Text } from "@/components/Text"
 import type { RosterStackParamList } from "@/navigators/DashboardNavigator"
+import { useTeamShifts } from "@/services/hooks/useTeamShifts"
 import { useAppTheme } from "@/theme/context"
 import { $styles } from "@/theme/styles"
 
 type Props = NativeStackScreenProps<RosterStackParamList, "TeamRoster">
 
 export function TeamRosterScreen(_props: Props) {
   const { themed } = useAppTheme()
+  const { teamShifts, isPending, error } = useTeamShifts(1, 1, 2025, "AM")
```

Comment:
To display an actual event
```suggestion
  const { teamShifts, isPending, error } = useTeamShifts(20, 0, 2025, "AM")
```